kubelet/datadog_checks/kubelet/prometheus.py

```diff
@@ -247,10 +244,10 @@ def _process_container_rate(self, metric_name, message):
         metrics = self._sum_values_by_context(message, self._get_container_id_if_container_metric)
         for c_id, metric in metrics.iteritems():
             pod_uid = self._get_pod_uid(metric.label)
-            if self.container_filter.is_excluded(c_id, pod_uid):
+            if self.pod_list_utils.is_excluded(c_id, pod_uid):
                 continue
 
-            tags = get_tags('docker://%s' % c_id, True)
+            tags = get_tags(c_id, True)
```

Comment:
nit: `get_tags` and `tags_for_container` are identical, should we move all their calls to a single one of them for consistency? Either only import `get_tags` in `common.py` or direct use it instead of the wrapper?

Reply:
Ended up using `get_tags` directly and removing `tags_for_container`